gemcalibration/src/common/CalibrationWeb.cc

```diff
@@ -0,0 +1,500 @@
+ *out << " <input type=\"text\" class=\"form-control\"  placeholder=\"0x000\" id=\"ohMask\">"<< std::endl;
+ *out << " </div>"<< std::endl;
+ *out << "</form>"<< std::endl;
+ *out << "</td> "<< std::endl;
+
+ *out << " <td>"<< std::endl;
+ *out << "<form>"<< std::endl;			 
+ *out << " <div class=\"form-group\">"<< std::endl;
+ *out << " <label for=\"VFatMask\">VFat Mask</label>"<< std::endl;
+ *out << " <input type=\"text\" class=\"form-control\" placeholder=\"path to mask\" id=\"VFatMask\">"<< std::endl;
+ *out << " </div>"<< std::endl;
+ *out << " </form>"<< std::endl;
+ 
+ *out << " </td> "<< std::endl;
+
+ *out << " <td>"<< std::endl;
+ *out << "<form>"<< std::endl;	
+ *out << "<form>"<< std::endl;			 
+ *out << " <div class=\"form-group\">"<< std::endl;
+ *out << " <label for=\"VFatChMin\">VFATChMin</label>"<< std::endl;
+ *out << " <input type=\"int\" class=\"form-control\" placeholder=\"Min Ch to scan:e.g. 0\" id=\"VFatChMin\">"<< std::endl;
+ *out << " </div>"<< std::endl;
+ *out << " </form>"<< std::endl;
+ *out << " </td> "<< std::endl;
+ 
+ *out << " <td>"<< std::endl;
+ *out << "<form>"<< std::endl;			 
+ *out << " <div class=\"form-group\">"<< std::endl;
+ *out << " <label for=\"VFatChMax\">VFATChMax</label>"<< std::endl;
+ *out << " <input type=\"int\" class=\"form-control\" placeholder=\"Max Ch to scan:e.g. 128\" id=\"VFatChMax\">"<< std::endl;
+ *out << " </div>"<< std::endl;
+ *out << " </form>"<< std::endl;
+ *out << " </td> "<< std::endl;
+ 
+ *out << " <td>"<< std::endl;
+ *out << "<form>"<< std::endl;			 
+ *out << " <div class=\"form-group\">"<< std::endl;
+ *out << " <label for=\"nEvts\">Number of events</label>"<< std::endl;
+ *out << " <input type=\"int\" class=\"form-control\" placeholder=\"100?\" id=\"nEvts\">"<< std::endl;
+ *out << " </div>"<< std::endl;
+ *out << " </form>"<< std::endl;
+ *out << " </td>"<< std::endl;
+ 
+ *out << " <td>"<< std::endl;
+ *out << "<form>"<< std::endl;			 
+ *out << " <div class=\"form-group\">"<< std::endl;
+ *out << " <label for=\"MSPL\">Pulse stretch</label>"<< std::endl;
+ *out << " <input type=\"text\" class=\"form-control\" placeholder=\"CFG_PULSE_STRETCH\" id=\"MSPL\">"<< std::endl;
+ *out << " </div>"<< std::endl;
+ *out << " </form>"<< std::endl;
+ *out << " </td>"<< std::endl;
+ 
+ *out << " <td>"<< std::endl;
+ *out << "<form>"<< std::endl;			 
+ *out << " <div class=\"form-group\">"<< std::endl;
+ *out << " <label for=\"Latency\"> Latency</label>"<< std::endl;
+ *out << " <input type=\"text\" class=\"form-control\" placeholder=\"CFG_LATENCY\" id=\"Latency\">"<< std::endl;
+ *out << " </div>"<< std::endl;
+ *out << " </form>"<< std::endl;
+ *out << " </td>"<< std::endl;
+ *out << " </tr> "<< std::endl;
+
+ *out << "  <tr> "<< std::endl;
+ *out << "  <td> "<< std::endl;
+ *out << "<div class=\"checkbox\">"<< std::endl;
+ *out << "<label>"<< std::endl;
+ *out << "<input type=\"checkbox\" class=\"check\"> eagleB" << std::endl;
```

Comment:
This line and all code below seems to be a carbon copy of the `eagleA` table definition above. Please refactor this using the `for` loop, looping over the CTP7's boards in question.
Also, the input fields should receive unique identifiers so that they can be distinguished in a callback function